packages/rust-bindings/ts-src/dapp.ts

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { Buffer } from 'buffer/index.js';
 
-import { initSync } from '../lib/dapp/web/esm';
+import { initSync } from '../lib/dapp/web/esm/index.js';
 import wasmBase64 from '../lib/dapp/web/esm/index_bg.wasm';
 
 // Expected to resolve to base64 encoded bytes of wasm module
 
 const bytes = Buffer.from(wasmBase64 as unknown as string, 'base64');
```

Comment:
Is it possible to remove the `as unknown as string` force cast now with the added `wasm.d.ts` declaration file?